phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css

```diff
@@ -122,6 +122,14 @@
     -fx-max-height: 2.5em;
 }
 
+#menuPane > .tab-header-area .tab:selected {
```

Comment:
Can you try 
#menuPane > .tab-header-area .tab:selected {
    -fx-background-color: linear-gradient(#529bda, #2173A5);
    -fx-focus-color: transparent;
    -fx-faint-focus-color: transparent;
}
instead?